packages/renderer/src/lib/editor/MonacoEditor.svelte

```diff
@@ -35,12 +35,31 @@ onMount(async () => {
     EditorSettings.SectionName + '.' + EditorSettings.FontSize,
   );
 
+  // check if we're in light or dark mode
+  const appearanceUtil = new AppearanceUtil();
+  const isDark = await appearanceUtil.isDarkMode();
+
+  // find the current terminal background color
+  const computedStyle = window.getComputedStyle(document.documentElement);
+  let bgColor = computedStyle.getPropertyValue('--pd-terminal-background').trim();
+
+  // Monaco only supports 6 char rgb values, so convert if we have 3 char format
+  if (bgColor?.length < 6) {
+    bgColor = bgColor
+      .split('')
+      .map(c => {
+        return c === '#' ? c : c + c;
+      })
+      .join('');
+  }
+
```

Comment:
should it be a function of appearanceUtil, to be able to get the value of a css property ?

Reply:
Yup, I think it will come up again. Since I have approvals here I'll do this via followup PR.

Reply:
#8264